static/templates/disbursements.hbs

```diff
@@ -45,6 +49,20 @@
 </div>
 
 <div class="panel__row">
+  <h4 class="panel__title">Filing information</h4>
+  <table>
+    <tr>
+      <td class="panel__term">Filing date</td>
+      <td>{{{ datetime receipt_date }}}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">View report page</td>
+      <td><a href="{{{ pdf_url }}}" target="_blank">View on fec.gov</a></td>
```

Comment:
I like "View report page", but then in the filings panels we'd need it to be "View report". Right now we use "View original image" for both.

I was looking at this and think it might make sense for the link under filing date to link to the full report, just so people can have it. I think we should also identify the report it was filed on by name, so you can see, e.g. April Quarterly 2015.